index.yaml

```diff
@@ -635,9 +635,9 @@ entries:
       catalog.cattle.io/type: cluster-tool
     apiVersion: v2
     appVersion: 0.1.2
```

Comment:
The `appVersion` in `packages/rancher-gpu-sharing/v0.1.0/Chart.yaml` needs to upgrade to `0.1.3`.